linq4j/src/test/java/org/apache/calcite/linq4j/test/Linq4jSuite.java

```diff
@@ -16,17 +16,12 @@
  */
 package org.apache.calcite.linq4j.test;
 
-import org.apache.calcite.linq4j.function.FunctionTest;
-import org.apache.calcite.linq4j.tree.TypeTest;
-
+import org.junit.platform.runner.JUnitPlatform;
+import org.junit.platform.suite.api.SelectClasses;
 import org.junit.runner.RunWith;
-import org.junit.runners.Suite;
 
-/**
- * Suite of all Linq4j tests.
- */
-@RunWith(Suite.class)
-@Suite.SuiteClasses({
+@RunWith(JUnitPlatform.class)
+@SelectClasses(
```

Comment:
I suggest we just drop Linq4jSuite.java file

Reply:
> I suggest we just drop Linq4jSuite.java file

ok, Let me remove it.